docs/running-gil-disabled.md

```diff
@@ -32,6 +32,12 @@ Python 3.13.0b1+ experimental free-threading build (heads/3.13:d93c4f9, May 21 2
 
 Verify that the GIL is disabled at runtime with the following incantation:
 
+```bash
+python -c "import sysconfig; print(bool(sysconfig.get_config_var('Py_GIL_DISABLED')))"
```

Comment:
the text above says this is a runtime check, but this checks how python was compiled

Reply:
I think that is correct? This new line is how we're checking this at runtime in NumPy, SciPy and everywhere else I can remember having to need it. I think it's the recommended method, since `_is_gil_disabled` is private.

Reply:
It's not a check of whether the GIL is actually currently disabled though. That's what I was trying to communicate with "GIL is disabled at runtime" since it can be re-enabled as well. Maybe add a sentence to clarify that the former checks the state of the build and is dtatic information but the `sys._is_gil_enabled()` can be `False` even on the free-threaded build.

Reply:
It's also the same information as you get from `Python -VV` and you can note that too, just more easily available from inside Python.

Reply:
Ah okay, I get what you meant now - makes sense, thanks.